docs/reference/release-notes/8.1.asciidoc

```diff
@@ -187,6 +204,11 @@ Machine Learning::
 * Set default value of 30 days for model prune window {es-pull}81377[#81377]
 * Track token positions and use source string to tag NER entities {es-pull}81275[#81275]
 * Warn when creating job with an unusual bucket span {es-pull}82145[#82145] (issue: {es-issue}81645[#81645])
+* Improve skip_model_update rule behavior {ml-pull}2096[#2096]
+* Upgrade Boost libraries to version 1.77 {ml-pull}2095[#2095]
```

Comment:
Can we make sure all the "Upgrade" items in this file go under the "Upgrades" section at the end? There are 3 ML items and 1 Security item to move.